tuning/trainercontroller/callback.py

```diff
@@ -250,7 +251,18 @@ def _take_control_actions(self, event_name: str, **kwargs):
                     continue
                 if rule_succeeded:
                     for operation_action in control_action.operation_actions:
-                        logger.info(
+                        log_levels = logging.get_log_levels_dict()
+                        trigger_log_level_code = logging.get_verbosity()
```

Comment:
Why do we care about the logging level set by default? Shouldn't the default be just `debug`?

Reply:
Addressed.